execute/plugin_tokendata_test.go

```diff
@@ -87,31 +100,32 @@ func Test_USDC_Transfer(t *testing.T) {
 	require.Len(t, outcome.CommitReports, 1)
 
 	// Round 3 - Filter
-	// Messages 102-104 are executed, 105 doesn't have token data ready
+	// Messages 102-104,106,108 are executed, 105 and 107 don't have token data ready
```

Comment:
108? I don't see that assertion in tests, seems like comment is wrong, isn't it?